data/locale/sv.ts

```diff
@@ -251,7 +251,7 @@ If you&apos;re interested in translating LMMS in another language or want to imp
     <name>AudioPortAudio::setupWidget</name>
     <message>
         <source>BACKEND</source>
-        <translation>BACKEND</translation>
+        <translation>BAKÄNDE</translation>
```

Comment:
Are you sure about this? It could be my lack of vocabulary but I don't think "Bakände" means backend. Imo borrowing from english sounds better here, but you're probably a better judge of which is more correct.